components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -12,21 +12,39 @@ const VulnerabilitiesGrid = ({ onUpdate }) => {
   const [grid, setGrid] = useState({ assets: {}, vulnerabilities: {} });
 
   const addItem = (obj, key, value) => {
-    obj[key] = value;
+    obj[key] = { name: value, data: {} };
```

Comment:
Is there a reason we went with an object for data here rather than an array? With there being multiple possible pieces of data we'd probably want an array of objects

Reply:
It was easier to construct objects like that, it can still have multiple pieces of data, but instead of it being:
_[{"id": "one", "value": "something"}, {"id":"two", "value": "something else" }]_
It would be:
_{"one": "something", "two": "something else" }_